Studio fix for Web Channels & grouping

tvgazza911 opened the pull request:
This fixes #9 by addressing the 2 issues described there;

1. Adding the studio for Web Channels - done in an elif statement as if both were present, the Network should be prioritised.

2. Country Code added to the studio name.

tvgazza911 committed:
Ensuring studio is set for shows airing on Web Channels and adding the country code to prevent incorrect grouping

romanvm commented on metadata.tvmaze/libs/data_utils.py:
```diff
@@ -165,8 +165,16 @@ def add_main_show_info(list_item, show_info):
-        video['studio'] = show_info['network']['name']
-        video['country'] = show_info['network']['country']['name']
+        country = show_info['network']['country']
+        video['studio'] = show_info['network']['name'] + ' (' + country['code'] + ')'
```
Please use string .format() method instead of concatenating. Also please use unicode literals for user-facing strings that may contain non-ASCII characters. 
See an example here: https://github.com/romanvm/kodi.tvmaze/blob/master/metadata.tvmaze/libs/actions.py#L42

tvgazza911 replied:
Changed

romanvm commented on metadata.tvmaze/libs/data_utils.py:
```diff
@@ -165,8 +165,16 @@ def add_main_show_info(list_item, show_info):
+        if show_info['webChannel']['country'] is not None:
+            country = show_info['network']['country']
+            video['country'] = country['name']
+            video['studio'] += ' (' + country['code'] + ')'
```
Same as above.

tvgazza911 replied:
Changed

romanvm reviewed (changes requested):
Thank you for your contribution. Please check my comments.

tvgazza911 committed:
Using unicode string format instead of concatenation and using correct property for Web Channel country

romanvm reviewed (approved):
Hopefully, it was tested :)

romanvm commented:
LGTM
